src/MSBuild/LiveLogger/ANSIBuilder.cs

```diff
@@ -196,11 +196,6 @@ public static string DoubleUnderlined(string text)
                 return String.Format("\x1b[21m{0}\x1b[24m", text);
             }
 
-            public static string Blinking(string text)
```

Comment:
Will we ever want blink anywhere else?

Reply:
I don't think I personally would . . .

Reply:
I can only imagine it in cases where user attention is needed. But as this is a logger (and not interactive) I can't think of any relevant scenario